org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -123,12 +123,6 @@
 					<type>jar</type>
 				</dependency>
 			</dependencies>
-			<repositories>

```

Comment:
Where do wie then find the org.eclipse.egit.github.core plugin which is used by org.eclipse.mylyn.github.core?

Reply:
My bad, fixed

Reply:
Will post soon an update after I have tried that https://download.eclipse.org/egit/github/updates as p2 works.

Reply:
Please reference the exact bundle as IU if you want to switch there

Reply:
And then please merge this one @BeckerFrank if it works for you

Reply:
> Will post soon an update after I have tried that https://download.eclipse.org/egit/github/updates as p2 works.

Sorry this did not work because the plugin org.eclipse.egit.github.core_6.1.0.202203080745-r.jar is not installable with  a separate feature. It is included in the feature org.eclipse.mylyn.github.feature_6.1.0.202203080745-r.jar

We need to request that org.eclipse.egit.github.core should be a standalone feature.

If we need to change the target file and remove the maven location I think we should try.
download the needed jar into a folder parallel to the target file and use Type Directory instead of Maven.
But for this case, I can't find a value for the path that works for a maven build and for the running eclipse installation.